src/GraphQL/Types/ISchema.cs

```diff
@@ -141,5 +141,20 @@ public interface ISchema
         /// access to the particular fields, you should use some authorization logic.
         /// </summary>
         ISchemaFilter Filter { get; set; }
+
+        /// <summary>
+        /// Returns a reference to the __schema introspection field available on the query graph type
+        /// </summary>
+        FieldType SchemaMetaFieldType { get; }
+
+        /// <summary>
+        /// Returns a reference to the __type introspection field available on the query graph type
+        /// </summary>
+        FieldType TypeMetaFieldType { get; }
+
+        /// <summary>
+        /// Returns a reference to the __typename introspection field available on any graph type
```

Comment:
_on any_ - no

http://spec.graphql.org/June2018/#sec-Type-Name-Introspection :
> GraphQL supports type name introspection at any point within a query by the meta‐field __typename: String! **when querying against any Object, Interface, or Union**.

Reply:
```suggestion
        /// Returns a reference to the __typename introspection field available on object, interface, or union graph types
```
How about that?

Reply:
OK

Reply:
on **any** object....  ?

Reply:
it's valid now, but that also works; need to change "types" to "type"

Reply:
OK